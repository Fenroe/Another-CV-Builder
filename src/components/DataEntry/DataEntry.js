import PersonalInfoEntry from "../PersonalInfoEntry/PersonalInfoEntry";
import StatementEntry from "../StatementEntry/StatementEntry";
import KeySkillsEntry from "../KeySkillsEntry/KeySkillsEntry";
import EmploymentHistoryEntry from "../EmploymentHistoryEntry/EmploymentHistoryEntry";
import EducationEntry from "../EducationEntry/EducationEntry";

function DataEntry() {
    return (
        <div>
            <PersonalInfoEntry />
            <StatementEntry />
            <KeySkillsEntry />
            <EmploymentHistoryEntry />
            <EducationEntry />
            <button>Preview</button>
        </div>
    )
}

export default DataEntry;