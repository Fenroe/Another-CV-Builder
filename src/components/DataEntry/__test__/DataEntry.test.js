import { render, cleanup } from "@testing-library/react";
import React from "react";
import "@testing-library/jest-dom";
import DataEntry from "../DataEntry";

afterEach(cleanup);

it("renders", () => {
    const {asFragment} = render(<DataEntry />);
    expect(asFragment()).toMatchSnapshot();
});
