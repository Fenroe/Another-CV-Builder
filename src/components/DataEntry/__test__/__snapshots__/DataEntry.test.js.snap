// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div>
    <div>
      <fieldset>
        <legend>
          Personal Details
        </legend>
        <label>
          First Name
        </label>
        <input />
        <label>
          Last Name
        </label>
        <input />
        <label>
          Email Adress
        </label>
        <input />
        <label>
          Phone Number
        </label>
        <input />
        <label>
          Website
        </label>
        <input />
      </fieldset>
    </div>
    <div>
      <fieldset>
        <legend>
          Statement
        </legend>
        <textarea />
      </fieldset>
    </div>
    <div>
      <fieldset>
        <legend>
          Key Skills
        </legend>
        <button>
          Add Skill
        </button>
      </fieldset>
    </div>
    <div>
      <fieldset>
        <legend>
          Employment History
        </legend>
        <button>
          Add Employer
        </button>
      </fieldset>
    </div>
    <div>
      <fieldset>
        <legend>
          Education
        </legend>
        <button>
          Add Institute
        </button>
      </fieldset>
    </div>
    <button>
      Preview
    </button>
  </div>
</DocumentFragment>
`;
